r/NEWS.md

```diff
@@ -17,6 +17,12 @@
   under the License.
 -->
 
+# arrow 9.0.0
+
+## Enhancements to date and time support
+
+* added `lubridate::qday()` (day of quarter)
```

Comment:
Could you  put this to `# arrow 8.0.0.9000` instead of creating `# arrow 9.0.0`?
`# arrow 8.0.0.9000` will be renamed to `# arrow 9.0.0` in our release process.

Reply:
I wasn't aware, thanks and done! :)